regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -70,9 +70,8 @@ <h4 class="preview-button" disabled>Building your PDF...</h4>
         {% include 'regulations/comment-additional-info.html' %}
 
         <div class="statement">
-          <p>You are filing a document into an official docket.
-          Any personal information entered on this form may be publicly viewable on the web.
-          </p>
```

Comment:
Would you mind popping this in the `comment-disclaimer.html`? Otherwise the checkbox below won't make sense for "base" installs